cogs/onmyoji.py

```diff
@@ -95,6 +101,59 @@ def generate_user_shikigami_locations(cls, shiki_name):
                 user_database_filtered_locations.append(temp_list)
         return user_database_filtered_locations
 
+class Embeds:
+    def shard_trading_embed(self, user):
+        need_list = ''
+        have_list = ''
+        for i in self.shard_trading_db[str(user.id)]['need']:
+            number, shiki = self.shard_split_variable(i)
+            need_list = f"{need_list}{bold(number)} {shiki}\n"
+        for i in self.shard_trading_db[str(user.id)]['have']:
+            number, shiki = self.shard_split_variable(i)
+            have_list = f"{have_list}{bold(number)} {shiki}\n"
+        if not user.nick:
+            nick = user.name
+        else:
+            nick = user.nick
+        if self.check_trading_status(user.id) == True:
+            trading_status = "available"
+        else:
+            trading_status = "unavailable"
+        try:
+            if self.shard_trading_db[str(user.id)]['notes']:
+                embed = discord.Embed(title=f"{nick}'s Shard Trading List", colour=discord.Colour(generate_random_color()), description=f"__**Notes:**__ {self.shard_trading_db[str(user.id)]['notes']}\n\nYou are **{trading_status}** for trading.")
```

Comment:
Could you restyle these? they're pretty hard to read
```suggestion
                embed = discord.Embed(
                    title=f"{nick}'s Shard Trading List",
                    colour=discord.Colour(generate_random_color()),
                    description=f"__**Notes:**__ {self.shard_trading_db[str(user.id)]['notes']}\n\nYou are **{trading_status}** for trading."
                )
```